app/views/cases/offender_sar/_data_requests.html.slim

```diff
@@ -1,9 +1,10 @@
 .grid-row
   .column-full.data-requests
-    h2.request--heading.data-requests__title
-      = t('cases.data_requests.index.heading')
+    - if case_details.current_state != 'rejected'
```

Comment:
Again, would it be better to hide this section entirely for rejected cases, rather than add all these conditionals?

Reply:
Yes I have removed these changes as no longer required.